torch/csrc/jit/codegen/cuda/tensor_view.cpp

```diff
@@ -446,12 +446,13 @@ TensorView* TensorView::reorder(const std::unordered_map<int, int>& old2new_) {
   return this;
 }
 
-/*
+/* TODO: should we remove this comment - having it on the declaration is enough?
```

Comment:
@csarofeen : do you prefer comments on declarations or definitions? (probably not both, since it's a recipe for getting out of sync and looking in the wrong place) For reference, Google code style prescribes comments on interfaces (declarations)

PS. is the correction in this comment valid?

Reply:
On declaration is fine. Probably a few places I need to clean that up.


I went through the one on the definition, there were a few errors.